qiita_db/analysis.py

```diff
@@ -705,64 +709,64 @@ def add_samples(self, samples):
         Parameters
         ----------
         samples : dictionary of lists
-            samples and the processed data id they come from in form
-            {processed_data_id: [sample1, sample2, ...], ...}
+            samples and the artifact id they come from in form
+            {artifact_id: [sample1, sample2, ...], ...}
         """
         with qdb.sql_connection.TRN:
             self._lock_check()
 
-            for pid, samps in viewitems(samples):
-                # get previously selected samples  for pid and filter them out
+            for aid, samps in viewitems(samples):
+                # get previously selected samples for aid and filter them out
                 sql = """SELECT sample_id
                          FROM qiita.analysis_sample
-                         WHERE processed_data_id = %s AND analysis_id = %s"""
-                qdb.sql_connection.TRN.add(sql, [pid, self._id])
+                         WHERE artifact_id = %s AND analysis_id = %s"""
+                qdb.sql_connection.TRN.add(sql, [aid, self._id])
                 prev_selected = qdb.sql_connection.TRN.execute_fetchflatten()
 
                 select = set(samps).difference(prev_selected)
                 sql = """INSERT INTO qiita.analysis_sample
-                            (analysis_id, processed_data_id, sample_id)
+                            (analysis_id, artifact_id, sample_id)
                          VALUES (%s, %s, %s)"""
-                args = [[self._id, pid, s] for s in select]
+                args = [[self._id, aid, s] for s in select]
                 qdb.sql_connection.TRN.add(sql, args, many=True)
                 qdb.sql_connection.TRN.execute()
 
-    def remove_samples(self, proc_data=None, samples=None):
+    def remove_samples(self, artifacts=None, samples=None):
         """Removes samples from the analysis
 
         Parameters
         ----------
-        proc_data : list, optional
-            processed data ids to remove, default None
+        artifacts : list, optional
+            Artifacts to remove, default None
         samples : list, optional
             sample ids to remove, default None
 
         Notes
         -----
         When only a list of samples given, the samples will be removed from all
```

Comment:
I know you didn't change this but are the black lines necessary? In the past we have use - at the beginning of each paragraph. Obviously, not blocking.